q2_diversity/_ordination.py

```diff
@@ -5,26 +5,111 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
-
+from sklearn.manifold import TSNE
 import skbio.stats.ordination
 import pandas as pd
+import umap
 
 
-def pcoa(distance_matrix: skbio.DistanceMatrix,
-         number_of_dimensions: int = None) -> skbio.OrdinationResults:
+def pcoa(
+    distance_matrix: skbio.DistanceMatrix, number_of_dimensions: int = None
+) -> skbio.OrdinationResults:
```

Comment:
There are a bunch of stylistic changes like this in this PR. I don't think they're necessary, and they add a bit of noise to the PR diff - can you please undo any of these changes that aren't actually implementing any new features, etc? Thanks!

Reply:
Thanks for addressing! To be clear, I wasn't requesting that you change anything about the new code contributed here ("added the old code style to new tsne and umap function"), I just wanted the broad-reaching stylistic changes reverted. It pollutes the diff, git history, and the git blame. You're welcome to introduce whatever style makes sense to you with the new code, we just ask that you not start changing _existing_ code, without a clear reason. Thanks!

Reply:
Yeah, the changes came from Flake8 which I installed on my code editor (Visual Studio Code). I had no intentions to change something from the old code."Old Style" was just a poor way to say "previous Style". I'm new to Git Hub and not so familiar with the correct Terms like WIP and how to adress commits probably so I'm very glad for every Feedback that I get